### ISSUE TITLE
Probleme auth

### ISSUE THREAD

**Issue by AzSiAz** [Hild-Franck/express-api-boilerplate#1]
Pour une meilleure sécurité il faut mettre une erreur générique sur la route de creation de nouveau compte et celle de connexion, comme ça le potentiel hacker ne sait pas si l'utilisateur ou le mail est deja utiliser et donc n'a pas d'info sur un potentiel compte

**Comment by Hild-Franck** [comment]
Effectivement, mais un utilisateur doit pouvoir savoir si son username ou mail est deja utilise, vu qu'ils doivent etre uniques

**Comment by AzSiAz** [comment]
Lui il doit pas savoir qu'ils sont uniques justement sinon une fois qu'il sait que l'email est utiliser il a juste a trouver le mot de passe alors que si il sait pas il doit quand même essayer de trouver le mot de passe en plus de continuer a chercher l'email.
Idem sur le mot de passe oublié, faut juste dire que le mail a été envoyer si l'email existe, pas que le mail n'existe pas
